biorxiv/pca_association_experiment/biorxiv_pca_plots.py

```diff
@@ -0,0 +1,639 @@
+# 
+# These PCs correspond to the following concepts:
+# 
+# | PC | Concept (negative vs positive)|
+# | --- | --- |
+# | PCA1 | Quantitative Biology vs Molecular Biology |
+# | PCA2 | Neuroscience vs Genomics |
+# | PCA3 | Sequencing vs Disease |
+# | PCA4 | Microbiology vs Cell Biology |
+# | PCA5 | RNA-seq vs Evolutional Biology | 
+
+# ### PCA1 vs PCA2
+
+# In[9]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_01_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+)
+
+
+# These word clouds depict the following concepts: quantitative biology vs molecular biology (left) and genomics vs neuroscience (right). The cells below provide evidence for the previous claim
+
+# In[10]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca1", y="pca2", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca01_v_pca02.png", dpi=500)
+print(g)
+
+
+# Take note that pca2 clusters neruoscience papers on the negative axis while genomics papers are on the positive axis (up and down). PCA 1 places papers that are more focused on quantitative biology on the right and molecular biology to the left. Hence why bioinforamtics papers are shifted more to the right and cell biology papers are shifted more to the left. This plot visually confirms the above finding.
+
+# ### PCA1 vs PCA 3
+
+# In[11]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_01_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_03_cossim_word_cloud.png'
+)
+
+
+# These word clouds depict the following concepts: quantitative biology vs molecular biology (left) and disease vs sequencing (right)
+
+# In[12]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca1", y="pca3", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca01_v_pca03.png", dpi=500)
+print(g)
+
+
+# Take note that pca3 clusters sequencing based papers on the negative axis (down) while disease papers are on the positive axis (up). When plotted against PCA1 it looks like epidemiology papers dominate the top right quadrant, while bottom right quadrant contains bioinformatics papers. This makes sense as many bioinformatic papers consist of some sort of sequencing technologies, while epidemiology is focused on measuring disease and its progression. Both take quantitative views, which is why they are on the positive size of PCA1.
+
+# ### PCA2 vs PCA3
+
+# In[13]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_03_cossim_word_cloud.png'
+)
+
+
+# These word clouds depict the following concepts: neuroscience to genomics (left) and disease vs sequencing (right)
+
+# In[14]:
+
+
+g = (
```

Comment:
The plots looks really interesting! I think your explanations make sense and are really clear.

I'm not sure if this would be helpful, but there are a lot of colors for each category on these UMAP plots, I wonder if you want to subsample just a few categories that you think are most representative to show the clustering pattern. Then you can start to add back in different categories to get a more clear picture of what is going on.